cloudinit/net/__init__.py

```diff
@@ -888,6 +948,8 @@ def get_interfaces(blacklist_drivers=None) -> list:
         driver = device_driver(name)
         if driver in blacklist_drivers:
             continue
+        if is_openvswitch_internal_interface(name):
+            continue
```

Comment:
Micro optimization, if you move this up above the driver = device_driver line, we can avoid looking up the device_driver of something that would be skipped.

Reply:
Good point; done.